README.md

```diff
@@ -1,3 +1,19 @@
 # PureScript-Cookbook
 
 An unofficial Cookbook for PureScript
+
+The recipes folder contains all the recipes in this repo in no particular order.
+
+## Contributing new recipes
+
+Follow these instructions for contributing new recipes:
+1. Pick an existing recipe to duplicate as a starting point. The `Template` recipe is probably the simplest".
```

Comment:
Before this step, there should be an "open an issue" or something. If we can specify in the issue what the module name and folder name will be, we can prevent naming clashes if two people happen to choose the same name for their recipes when working on their independent PRs.

Reply:
Good catch. I meant to comment about that. I think it's better to be lighter on the procedural requirements until we encounter this sort of problem.

Reply:
Covered by #11